voyager-publish/index.js

```diff
@@ -139,8 +139,12 @@ export const publish = async ({
     }
   }
 
-  logger.log('Vacuuming measurements')
-  await pgPool.query('VACUUM measurements')
+  if (totalCount > 1_000_000) {
+    console.log('Skipping `VACUUM measurements` - the table is too large')
```

Comment:
Ouch, this should have been `logger.log` 🙈 

Fixed in fb845bb (pushed directly to `main`).